### PR TITLE
Add installation docs for installing peerDeps

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by amilajack** [comment]
This seems to conflict with https://github.com/uber/react-digraph/pull/59

File: README.md [hunk 1]
```diff
@@ -6,10 +6,15 @@ A React component which makes it easy to create a directed graph editor without
...
 ## Installation
 
-```
+```bash
 npm install --save react-digraph
 ```
 
+If you don't have the following peerDependenies, make sure to install them:
+```bash
+npm install --save d3 radium react react-dom react-icons
```
Comment (ajbogh) [inline README.md]: We've made a change to master that now includes d3, radium, and react-icons as core dependencies for react-digraph. Can you remove d3, radium, and react-icons from the list?

**Comment by amilajack** [comment]
Done

**Comment by ajbogh** [comment]
Thanks!

**Pull request merged**